src/partner/modules/EventList/EventList.tsx

```diff
@@ -5,6 +5,7 @@ import { ListStyled } from "src/partner/modules/ui/";
 
 interface IEventListProps {
   events: IEvent[];
+  handleCancelEvent: Function;
```

Comment:
avoid using `Function`, if the function just takes any event a doesn't return anything use:

```ts
export interface IEventListContainerProps {
  handleCancelEvent: (e: any) => void;
}
```

else use a meaningful function signature